packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1014,6 +1065,27 @@ class RawScrollbar extends StatefulWidget {
   /// {@endtemplate}
   final bool? isAlwaysShown;
 
+  /// The [shape] of the scrollbar's thumb.
+  ///
+  /// If you need just a simple rectangle with rounded borders, use [radius].
```

Comment:
I think this could be stated a little more legalistically:

Only one of [radius] and [shape] may be specified. For a rounded rectangle, it's simplest to just specify [radius].  By default the scrollbar thumb's shape is a simple rectangle.